src/controllers/dashboard/streaming.html

```diff
@@ -7,7 +7,7 @@
                         <h2 class="sectionTitle">${TabStreaming}</h2>
                     </div>
                     <div class="inputContainer">
-                        <input is="emby-input" type="number" id="txtRemoteClientBitrateLimit" pattern="[0-9]*" min="0" step=".25" label="${LabelRemoteClientBitrateLimit}" />
+                        <input is="emby-input" type="number" id="txtRemoteClientBitrateLimit" inputmode="decimal" pattern="[0-9]*(\.[0-9]*)?" min="0" step=".25" label="${LabelRemoteClientBitrateLimit}" />
```

Comment:
```suggestion
                        <input is="emby-input" type="number" id="txtRemoteClientBitrateLimit" inputmode="decimal" pattern="[0-9]*(\.[0-9]+)?" min="0" step=".25" label="${LabelRemoteClientBitrateLimit}" />
```

`1.0` should be valid
`1.` should be invalid